packages/dataviews/src/dataviews-layouts/grid/preview-size-picker.tsx

```diff
@@ -48,10 +48,12 @@ export default function PreviewSizePicker() {
 
 	const layoutPreviewSize = view.layout?.previewSize ?? 230; // Default to the third smallest size if no preview size is set.
 	// If the container has resized and the set preview size is no longer available,
-	// we reset it to the next smallest size.
+	// we reset it to the next smallest size, or the smallest available size.
 	const previewSizeToUse = breakValues
 		.map( ( size, index ) => ( { ...size, index } ) )
-		.filter( ( size ) => size.value <= layoutPreviewSize )
+		.filter(
+			( size ) => size.index === 0 || size.value <= layoutPreviewSize
+		)
```

Comment:
How about checking whether the result is available before accessing it's value. Would it be clearer to handle the error 🤔

```js
[ 0 ]?.index ?? breakValues[ 0 ].index
```